config/charts/inferencepool/templates/epp-deployment.yaml

```diff
@@ -53,12 +53,12 @@ spec:
         livenessProbe:
           grpc:
             port: 9003
-            service: inference-extension
+            service: envoy.service.ext_proc.v3.ExternalProcessor
```

Comment:
what does this represent, would it matter if we remove it?

Reply:
I guess this should match the name in the health check routine, otherwise we return `HealthCheckResponse_SERVICE_UNKNOWN`. 

Wondering why we want to take a dependency on what the service name as defined in https://github.com/envoyproxy/go-control-plane/blob/main/envoy/service/ext_proc/v3/external_processor_grpc.pb.go#L135 when we could control that on our end?

Reply:
To my understanding the gRPC service might not be able to change because ext_proc expects this specific one by name to be present for its requests. At least by default, maybe that's configurable somewhere (on the Envoy side)? In any case, this is the actual service name we've been using since the beginning: if you turn on reflection on the current gRPC server and do a list, this is the exposed service that Envoy is expecting.